com.projecthawkthorne.Hawkthorne.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: com.projecthawkthorne.Hawkthorne
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: Hawkthorne.sh
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+
+modules:
+  - name: love2d
+    sources:
+    - type: archive
+      url: https://github.com/love2d/love/releases/download/11.5/love-11.5-linux-src.tar.gz
+      sha256: 066e0843f71aa9fd28b8eaf27d41abb74bfaef7556153ac2e3cf08eafc874c39
+    modules:
+      - 'shared-modules/luajit/luajit.json'
+      - name: libmodplug
+        sources:
+        - type: archive
+          url: https://downloads.sourceforge.net/project/modplug-xmms/libmodplug/0.8.9.0/libmodplug-0.8.9.0.tar.gz
+          sha256: 457ca5a6c179656d66c01505c0d95fafaead4329b9dbaa0f997d00a3508ad9de
+
+  - name: hawkthorne
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/hawkthorne/hawkthorne-journey/releases/download/v1.1.0/hawkthorne.love.zip
+        sha256: f9385243121b85fa8bd066feb9a8952efab6e6152f03c2f616a48fe4855ffa61
+      - type: file
+        path: Hawkthorne-launcher.sh
+      - type: file
+        url: https://github.com/hawkthorne/hawkthorne-journey/raw/refs/heads/master/templates/linux/love.desktop
```

Comment:
this shouldn't reference the tip of a branch because it will break the build as soon as it changes.